### PR TITLE
I18n Support for Patchouli Book and Optimization of CHS Translation

### PR DESCRIPTION
**Pull request by WuzgXY-GitHub**

### CODE REVIEW

File: TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang [hunk 1]
```diff
@@ -288,3 +297,6 @@ tile.theaurorian.weepingwillowlog.name=垂柳原木
 tile.theaurorian.weepingwillowplanks.name=垂柳木板
 tile.theaurorian.weepingwillowplanksstairs.name=垂柳木楼梯
 tile.theaurorian.weepingwillowsapling.name=垂柳树苗
+
+patchouli.theaurorian.guide.name=极光幽境指导
+patchouli.theaurorian.guide.desc=关于本模组一切你需要知道的事！
```
Comment (Determancer) [inline TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang]:
```suggestion
patchouli.theaurorian.guide.desc=关于本模组你所需要知道的一切！
```

File: TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang [hunk 2]
```diff
@@ -288,3 +297,6 @@ tile.theaurorian.weepingwillowlog.name=垂柳原木
 tile.theaurorian.weepingwillowplanks.name=垂柳木板
 tile.theaurorian.weepingwillowplanksstairs.name=垂柳木楼梯
 tile.theaurorian.weepingwillowsapling.name=垂柳树苗
+
+patchouli.theaurorian.guide.name=极光幽境指导
```
Comment (TUsama) [inline TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang]: |Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

“指导”作名词时，一般指的是“指导的动作”，这里可以改成“指南”“手册”一类的

**Review by TUsama — CHANGES_REQUESTED** [review]

**Review by TUsama — APPROVED** [review]

File: TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang [hunk 3]
```diff
@@ -128,18 +132,25 @@ item.theaurorian.umbrapickaxe.name=本影镐
...
 modifier.aurorianempowered.name=极光之赐
-modifier.conarmmoonlit_armor.desc=§o它是个夜猫子！§r\n在夜晚有50%的机会不消耗耐久，但在白天有50%的机会消耗更多耐久！
+modifier.conarmmoonlit_armor.desc=§o它是个夜猫子！§r\n在夜晚有 50% 的机会不消耗耐久，但在白天有 50% 的机会消耗更多耐久！
 modifier.conarmmoonlit_armor.name=月辉
-modifier.tamoonlit.desc=§o它是个夜猫子！§r\n在夜晚有50%的机会不消耗耐久，但在白天有50%的机会消耗更多耐久！
+modifier.tamoonlit.desc=§o它是个夜猫子！§r\n在夜晚有 50% 的机会不消耗耐久，但在白天有 50% 的机会消耗更多耐久！
 modifier.tamoonlit.name=月辉
+
+patchouli.theaurorian.guide.desc=关于本模组你所需要知道的一切！
+patchouli.theaurorian.guide.name=极光导书
```
Comment (Determancer) [inline TheAurorian/src/main/resources/assets/theaurorian/lang/zh_cn.lang]: 导书是什么鬼
```suggestion
patchouli.theaurorian.guide.name=极光幽境指南
```

**Comment by WuzgXY-GitHub** [comment]
Sorry for creating a branch later than I should, and I am going to close this pull request.

**Pull request closed without merging**